zone/gm_commands/npcspawn.cpp

```diff
@@ -9,7 +9,18 @@ void command_npcspawn(Client *c, const Seperator *sep)
 
 	int arguments = sep->argnum;
 	if (!arguments) {
-		c->Message(Chat::White, "Command Syntax: #npcspawn [Add|Create|Delete|Remove|Update]");
+		c->Message(Chat::White, "Command Syntax: #npcspawn [Add|Create|Delete|Remove|Update|Clone|Help] [optional 3rd parameter]");
+		return;
+	}
+
+	if (!strcasecmp(sep->arg[1], "help")) {
+		c->Message(Chat::White, "Command Syntax: #npcspawn [Add|Create|Delete|Remove|Update|Clone|Help] [optional 3rd parameter]");
+		c->Message(Chat::White, "Add: Using the same targeted NPC ID, creates new spawn2 and spawngroup entries [3rd parameter: respawntime]");
+		c->Message(Chat::White, "Create: Creates new NPC type copying the data from the targeted NPC, with new spawn2 and spawngroup entries [3rd parameter: respawntime]");
+		c->Message(Chat::White, "Delete: Deletes the spawn2, spawngroup, spawnentry and npc_types rows for targeted NPC");
+		c->Message(Chat::White, "Remove: Deletes the spawn2 row for targeted NPC [3rd parameter: also delete spawngroup and spawnentry rows if > 0]");
+		c->Message(Chat::White, "Update: Updates NPC appearance in database");
+		c->Message(Chat::White, "Clone: Copies targeted NPC and spawngroup, creating only a spawn2 entry [3rd parameter: respawntime]");
 		return;
 	}
```

Comment:
May want to do something like `#npcedit` does.

https://github.com/EQEmu/Server/blob/master/zone/gm_commands/npcedit.cpp#L1709-L1816

Reply:
Resolved and tested in new [commit](https://github.com/EQEmu/Server/pull/4311/commits/6524e60ba154786a0f7a5727e0717644fbd6a1d3)